conformance_pack/vpc.sp

```diff
@@ -10,10 +10,12 @@ control "vpc_flow_logs_enabled" {
   sql         = query.vpc_flow_logs_enabled.sql
 
   tags = merge(local.conformance_pack_vpc_common_tags, {
-    hipaa              = "true"
-    nist_800_53_rev_4  = "true"
-    nist_csf           = "true"
-    rbi_cyber_security = "true"
+    cis_foundational_item_id = "2.9"
```

Comment:
Why is this tag being added here? I'd only expect to see `gdpr` being added in this PR